packages/flutter_tools/lib/src/commands/analyze_continuously.dart

```diff
@@ -169,6 +195,13 @@ class AnalysisServer {
       '--sdk',
       sdk,
     ];
+    // Let the analysis server know that the flutter repository is being analyzed
+    // so that it can enable the public_member_api_docs lint even though
+    // the analysis_options file does not have that lint enabled.
```

Comment:
Several times Brian and I have discussed the best approach here. Our current top prospect is to add support for linter summaries in `analysis_options.yaml`, but that will take a while. This command line flag is intended to be very specific for your needs until the more general purpose mechanism is available.